test/colorization/results/deep-nested.FULL-SCOPE.json.txt

```diff
@@ -1,13 +1,13 @@
 TEST STRING: "['test1']"
```

Comment:
The remaining changes are test updates to match the changes. No reason to review, as long as they pass...